### ISSUE TITLE
Accr

### ISSUE THREAD

**Issue by barbarahartel** [carbon-design-system/carbon-components-vue#1087]
I use the CVUiShell component in VueJS and on this site you mention that the violations against Accessibility Guidelines are identified as high priority issues.
Do you have a timeline when these issues will be fixed?

**Comment by barbarahartel** [comment]
Do you have any idea when Accessability support is available? 
Please let me know it since I have to solve the issues in my application if nothing is provided by Carbon.

**Comment by BryceBarbara** [comment]
Maybe I missed something, but what are the issues that you've found?

**Comment by davidnixon** [comment]
This was originally written against Vue 2 components but is a good candidate for an accessibility test for the `src/components/CvUIShell/CvHeaderMenuButton.vue`  component in Vue 3. See [guidance](https://github.com/carbon-design-system/carbon-components-vue/blob/main/ACCESSIBILITY-TESTING.md)
